### PR TITLE
Fixed compilation error due to missing parameter in __using__

### PR DESCRIPTION
**Pull request by johnf9896**
Hello.
After adding the library to my project it couldn't compile in the `use Loki` part. 
This is because the `use` requires the macro `__using__/1` to be declared and not `__using__/0`.
I just added an unused parameter `_` to the macro for it to compile correctly.

### CODE REVIEW

**Comment by khusnetdinov** [comment]
Hello, @johnf9896! This is your first Pull Request that will be reviewed by [Ebert](https://ebertapp.io/), an automatic Code Review service. It will leave comments on this diff with potential issues and style violations found in the code as you push new commits. You can also see all the issues found on this Pull Request on its [review page](https://ebertapp.io/github/khusnetdinov/loki/pulls/39). Please check our [documentation](https://docs.ebertapp.io) for more information.

**Comment by khusnetdinov** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 possible new issue (including those that may have been commented here).
* 1 fixed issue! 🎉

You can see more details about this review at https://ebertapp.io/github/khusnetdinov/loki/pulls/39.

**Comment by khusnetdinov** [comment]
@johnf9896, Could you provide versions of deps in the application? Locally it compiles properly.
Please, could you open issue for your question about parser options and remove from this PR - it relates to another subject not in this PR. And Please not fix README spacesб, just rebase - fixed it.

File: README.md [hunk 1]
```diff
@@ -20,12 +20,13 @@ end
 
 ## Usage
 
-Import all or desired module to scope:
+Import all or desired modules to scope:
 
 ```elixir
 defmodule Project do
-  use Loki
+  use Loki # Import all modules
```
Comment (khusnetdinov) [inline README.md]: Put this comment above:
```elixir
  # Import all modules
  use Loki 
```

File: README.md [hunk 2]
```diff
@@ -40,61 +41,66 @@ end
 ## Modules
 
 ### Loki.Shell
+
```
Comment (khusnetdinov) [inline README.md]: Do not update spaces, just rebase from master. I fixed it.

**Comment by khusnetdinov** [comment]
@johnf9896 , Could you please remove from issue question about options parse and create the issue - this PR has other subjects. I tried to compile library locale and it works fine. Could you please provide me versions of your applications, elixir, and mix and other if others if need. Need to understand where the problem.

**Comment by khusnetdinov** [comment]
@johnf9896 , please rebase brunch.

**Comment by johnf9896** [comment]
I am using Elixir 1.7.3

**Comment by khusnetdinov** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 possible new issue (including those that may have been commented here).
* 1 fixed issue! 🎉

But beware that this branch is **[2 commits behind the `khusnetdinov:master` branch](https://github.com/khusnetdinov/loki/compare/johnf9896:master...khusnetdinov:master)**, and a review of an up to date branch would produce more accurate results.

You can see more details about this review at https://ebertapp.io/github/khusnetdinov/loki/pulls/39.

**Comment by johnf9896** [comment]
It's my project what doesn't compile, not the library. It happens when I add `use Loki` because it says it needs `__using__/1`

**Pull request merged**